### ISSUE TITLE
Remove score indicator 

### ISSUE THREAD

**Issue by Marcel510** [openscope/openscope#1020]
As already said on Slack, the score indicator should be removed  from the sim since it doesn't really suit a simulator.

**Comment by odiferousmint** [comment]
I agree. The stats do not make much sense, especially not with these numbers. If it does not get removed (because some people do like stats), then perhaps there should be a setting to turn it on or off and the numbers should be adjusted as you get a pretty high penalty for most if not all mistakes, but not much reward for successful departures, landings and whatnot.

**Comment by erikquinn** [comment]
Agreed. Elevating the priority so we can get rid of the silliness. :)
